docs/release-notes/release-notes-0.17.0.md

```diff
@@ -154,6 +154,7 @@
   latest version of `google.golang.org/protobuf` instead of the deprecated
   `github.com/golang/protobuf/jsonpb`
   module](https://github.com/lightningnetwork/lnd/pull/7659).
+* [Remove Litecoin code](https://github.com/lightningnetwork/lnd/pull/7867)
```

Comment:
Should be 18 now, and we should also mention the deprecated config fields.

Reply:
done :)